src/tactic/linarith.lean

```diff
@@ -282,21 +282,26 @@ match c1.keys, c2.keys with
 | _, _ := none
 end
 
+meta def list.mfind {α} (tac : α → tactic unit) : list α → tactic α

```

Comment:
I'm not sure if `list.mfind` or `rb_map.find_defeq` are generally useful. Right now they're in the `linarith` namespace. I could move (and rename?) them if people think it's worth it.